src/main/scala/dpla/ingestion3/mappers/providers/FlMapping.scala

```diff
@@ -0,0 +1,127 @@
+package dpla.ingestion3.mappers.providers
+
+import dpla.ingestion3.enrichments.normalizations.filters.ExtentIdentificationList
+import dpla.ingestion3.mappers.utils._
+import dpla.ingestion3.messages.IngestMessageTemplates
+import dpla.ingestion3.model.DplaMapData.{AtLeastOne, ExactlyOne, ZeroToMany, ZeroToOne}
+import dpla.ingestion3.model._
+import dpla.ingestion3.utils.Utils
+import org.json4s
+import org.json4s.JsonDSL._
+import org.json4s._
+
+
+class FlMapping extends JsonMapping with JsonExtractor with IngestMessageTemplates {
+
+  val formatBlockList: Set[String] = ExtentIdentificationList.termList
+
+  // ID minting functions
+  override def useProviderName: Boolean = true
+
+  override def getProviderName: String = "florida"
+
+  override def originalId(implicit data: Document[JValue]): ZeroToOne[String] =
+    extractStrings(unwrap(data) \ "isShownAt").headOption
+
+  // OreAggregation
+  override def dataProvider(data: Document[JValue]): ZeroToMany[EdmAgent] =
+    extractStrings(unwrap(data) \ "dataProvider").map(nameOnlyAgent)
+
+  override def dplaUri(data: Document[JValue]): ZeroToOne[URI] = mintDplaItemUri(data)
+
+  override def edmRights(data: Document[json4s.JValue]): ZeroToMany[URI] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "rights" \ "@id")
+      .filter(_.nonEmpty) // FIXME filtering non-empty values should be a standard edmRights normalization
+      .map(URI)
```

Comment:
Calling out the `FIXME` in case you had intended to fix it for this PR

Reply:
Thanks -- I think that is for another PR. Just identifying some additional future work ;)